pkg/parser/parser.go

```diff
@@ -93,7 +95,7 @@ func (c *Parser) SupportedExtensions() model.Extensions {
 func validateArguments(types, validArgs []string) error {
 	validArgs = removeDuplicateValues(validArgs)
 	if invalidType, ok, _ := contains(types, validArgs); !ok {
-		return fmt.Errorf(fmt.Sprintf("Unknown Argument: %s\nValid Arguments:\n  %s\n", invalidType, strings.Join(validArgs, "\n  ")))
+		return fmt.Errorf(fmt.Sprintf("unknown Argument: %s\nvalid Arguments:\n  %s\n", invalidType, strings.Join(validArgs, "\n  ")))
```

Comment:
shouldn't argument(s) be in lower case too?

Reply:
solved